script_vts.py

```diff
@@ -202,95 +207,84 @@ def formating_file():
             out_file.truncate()
     except IOError as strip_err:
         exception_handler(True, strip_err)
-        exit(1)
```

Comment:
Why has this been removed?

Reply:
Sorry my bad I put it again. Probably during one of the format I did.